src/app/launcher/create-app/mission-runtime-createapp-step/mission-runtime-createapp-step.component.spec.ts

```diff
@@ -120,4 +120,228 @@ describe('MissionRuntimeStepComponent', () => {
+    fixture.detectChanges();
+    element = fixture.nativeElement;
+    let runtimesSection = element.querySelectorAll('.card-pf-body')[1];
+    let runtimeGroupItem = runtimesSection.querySelector('.list-group-item');
+    let radioBtn = <HTMLInputElement>runtimeGroupItem.querySelector('input[type="radio"]');
+    let beforeChange = runtimeGroupItem.classList.contains('selected-list-item');
+    radioBtn.click();
+    tick();
+    fixture.detectChanges();
+    expect(beforeChange === false && runtimeGroupItem.classList.contains('selected-list-item') === true).toBeTruthy();
+  }));
+
+  it('should update the selected runtime in launcher component', fakeAsync(() => {
+    fixture.detectChanges();
+    element = fixture.nativeElement;
+    let runtimesSection = element.querySelectorAll('.card-pf-body')[1];
+    let runtimeGroupItem = runtimesSection.querySelector('.list-group-item');
+    let radioBtn = <HTMLInputElement>runtimeGroupItem.querySelector('input[type="radio"]');
+    let beforeChange = runtimeGroupItem.classList.contains('selected-list-item');
+    radioBtn.click();
+    tick();
+    fixture.detectChanges();
+    expect(component.launcherComponent.summary.runtime).toBe(component.runtimes[0]);
+  }));
+
+  it('should have the suggested runtimes tag when runtime.suggested field is true', () => {
+    element = fixture.nativeElement;
+    let runtimesSection = element.querySelectorAll('.card-pf-body')[1];
+    let featuredTag = runtimesSection.querySelector('.f8launcher-tag-featured');
+    expect(featuredTag).toBeDefined();
+  });
+
+  it('should not have the suggested runtimes tag when runtime.suggested field is false', () => {
+    component.missions[0].suggested = false;
+    fixture.detectChanges();
+    element = fixture.nativeElement;
+    let runtimesSection = element.querySelectorAll('.card-pf-body')[1];
+    let featuredTag = runtimesSection.querySelector('.f8launcher-tag-featured');
+    expect(featuredTag).toBeNull();
+  });
+
+  it('should have the runtime name as specified', () => {
+    element = fixture.nativeElement;
+    let runtimesSection = element.querySelectorAll('.card-pf-body')[1];
+    let runtimesHead = runtimesSection.querySelectorAll('.list-group-item-heading');
+    let runtimes = component.runtimes, len = runtimes.length;
+    for (let i = 0; i < len; ++ i) {
+      expect((<HTMLDivElement>runtimesHead[i]).innerText).toBe(runtimes[i].name);
+    }
+  });
+
+  it('should have the runtime description as specified', () => {
+    element = fixture.nativeElement;
+    let runtimesSection = element.querySelectorAll('.card-pf-body')[1];
+    let runtimesHead = runtimesSection.querySelectorAll('.list-group-item-text');
+    let runtimes = component.runtimes, len = runtimes.length;
+    for (let i = 0; i < len; ++ i) {
+      expect((<HTMLDivElement>runtimesHead[i]).innerText).toBe(runtimes[i].description);
```

Comment:
This is failing with:

Expected 'Brief description of the technology...
More' 
to be 'Brief description of the technology...'

The description is truncated in the UI after so many chars and a 'more' link is added. If you shorted your mock description, the 'more' link should not be present.